client/js/socket-events/configuration.js

```diff
@@ -176,6 +176,17 @@ function parseOverrideParams(params, data) {
 			continue;
 		}
 
+		// When the network is locked, URL overrides should not affect disabled fields
+		if (data.lockNetwork &&
+				["host", "port", "tls", "rejectUnauthorized"].includes(key)) {
+			continue;
+		}
+
+		// When the network is not displayed, its name in the UI is not customizable
```

Comment:
I believe this is not enforced on the server.

Reply:
Mmmh, indeed. Anything (simple if possible) I need to add to this PR? Or for later?
It's plain weird that we can change the name on the title from the URL when not displayed (and a regression) but I see what you're saying.

Reply:
Hm. I think this `if` you added isn't even needed because you shouldn't have added `name` on the server in `getClientConfiguration`.

Server defaults to defaults.name if none provided here and `displayNetwork ` is false: https://github.com/thelounge/thelounge/blob/master/src/client.js#L204

Technically it's still possible to directly send a name by modifying the form in devtools, but overriding the URL shouldn't work?